packages/web3-rpc-methods/src/eth_rpc_methods.ts

```diff
@@ -577,6 +578,44 @@ export async function createAccessList(
 	});
 }
 
+export async function sendUserOperation(
+	requestManager: Web3RequestManager,
+	userOperation: UserOperation,
+	entryPoint: Address,
+) {
```

Comment:
validation/init is missing, can you add validation before `send` as mentioned in EIP :
" All fields MUST be set as hex values. empty bytes block (e.g. empty initCode) MUST be set to "0x"".

for example you can see other functions doing validation in this file.

Reply:
I added the [isUserOperationAllHex](https://github.com/web3/web3.js/pull/6302/commits/64f698bb095b182e3f4fc3c29d069a355dbb1241#diff-ded109291382e85eaff43bc144de95df86194f132993f86d28ab0cde62018e65) method, but now I'm concerned that all the data users input will be in hex values?!